src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -4102,74 +4100,7 @@ public void WriteCommentsInArray(bool formatted, bool skipValidation, string com
 
                 WriteCommentValue(jsonUtf8, i, comment);
 
-                jsonUtf8.WriteStartArray();
-                jsonUtf8.WriteEndArray();
-
-                WriteCommentValue(jsonUtf8, i, comment);
-
-                jsonUtf8.WriteStartObject();
-                jsonUtf8.WriteEndObject();
-
-                WriteCommentValue(jsonUtf8, i, comment);
-
-                jsonUtf8.WriteEndArray();
-
-                WriteCommentValue(jsonUtf8, i, comment);
-
-                jsonUtf8.Flush();
-
-                JsonTestHelper.AssertContents(expectedStr, output);
-            }
-        }
-
-        [Theory]
-        [InlineData(true, true, "comment")]
```

Comment:
It would be nice to keep these tests as well, complementing the string literal tests.

Reply:
Added back.